nucliadb/nucliadb/reader/api/v1/services.py

```diff
@@ -313,19 +314,28 @@ async def get_configuration(request: Request, kbid: str):
     f"/{KB_PREFIX}/{{kbid}}/notifications",
     status_code=200,
     name="Knowledge Box Notifications Stream",
-    description="Provides a stream of activity notifications for the given Knowledge Box.",
+    description="Provides a stream of activity notifications for the given Knowledge Box. The stream will be automatically closed after 2 minutes.",  # noqa: E501
```

Comment:
If this value depends on istio or something related to our cloud infrastructure, a comment here (for us) will be useful

Reply:
yup, I've added a comment somewhere else in the PR